content/master/software/install.md

```diff
@@ -251,6 +251,7 @@ at the table below.
 | Alpha | `--enable-external-secret-stores` | Enable support for External Secret Stores. |
 | Alpha | `--enable-usages` | Enable support for Usages. |
 | Alpha | `--enable-realtime-compositions` | Enable support for real time compositions. |
+| Alpha | `--enable-claim-ssa` | Enable support for using server-side apply to sync claims with XRs. |
```

Comment:
We perform field manager migration for claims created by previous versions of crossplane. This migration is irreversible, i.e. once performed by turning the flag on, it cannot be reverted by turning the flag off.

Reply:
Would it require a reinstalling Crossplane to reset?

Reply:
reinstalling Crossplane would not help - the data are affected. The best bet would be to make a backup before turning the flag on. The backup solution should be able to restore `managedFields` properly, in case when the we want to restart Crossplane with the flag off.

Reply:
Tested this out and in my scenario it worked fine disabling the flag.
I did the following:
- installed with --enable-ssa-claim
- deployed an XRC with a map field with keys “foo” and “bar”
- removing “foo” from the XRC did remove it from the XR as expected
- added back “foo” to the XRC
- reinstalled without --enable-ssa-claim
- tried removing foo from the XRC, foo was NOT removed from the XR as expected
- added something to the map in the XRC, which was correctly propagated to the XR
Did I miss something?

Reply:
My comment was motivated by https://github.com/kubernetes/client-go/blob/master/util/csaupgrade/upgrade.go#L61-L80.  @phisco I have tried the same steps, and noticed that ssa manager remains after reinstalling without the flag. Not sure if this can produce new issues.

Reply:
Would it make sense to address this by adding a note to `claim-server-side-apply.md` that if you disable the feature flag, Crossplane won't clean up SSA field managers? We could link to upstream docs about what field managers are.

It sounds like leaving SSA field managers might have no meaningful impact though, so not sure how valuable this is document.

Reply:
> It sounds like leaving SSA field managers might have no meaningful impact though, so not sure how valuable this is document.

It is really hard to judge about the implications, because fields in a composite will be still owned by now not existing manager.

Reply:
I don't totally follow this conversation but it sounds like if ssa is enabled there are some things that can't be undone, right? 

I just want to make sure we call this out. We don't need to make the reader nervous but something like "Enabling SSA is a permanent change to your Kubernetes cluster and can only be undone by <uninstalling crossplane?>"